src/components/inspectors/PreviewPanel.vue

```diff
@@ -120,8 +154,25 @@ export default {
     onSelectedPreview(item) {
       this.selectedPreview = item;
     },
-    previewNode(node) {
-      this.taskTitle = node?.name;
+    previewNode(force = false) {
+      if (!this.highlightedNode || (!this.visible && !force)) {
+        return;
+      }
+
+      const previewConfig = this.previewConfigs.find(config => {
+        return config.matcher(this.data);
+      });
+
+      var clone = {};
```

Comment:
don't use var anymore, let if you are going to override the value of the variable later on
```suggestion
      let clone = {};
```